### PR TITLE
V3 - Organize Studio Folder Structure

### PR DESCRIPTION
**Pull request by anemne**
## Short Description

Cleanup in studio structure to better match deskstrukture. 

## Visual Overview (Image/Video)

- Created sitesettingsfolder under documents 
- renamed redirect file to brokenLinks
- Moved pagebuilder to documents 
Screenshot 2024-09-17 at 15 14 30

- Moved queries from page.ts to own file for compensations
Screenshot 2024-09-17 at 15 15 57

Renamed company information --> parent company 
Screenshot 2024-09-17 at 15 23 44

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -8044,7 +8044,6 @@
       "resolved": "https://registry.npmjs.org/@typescript-eslint/eslint-plugin/-/eslint-plugin-8.6.0.tgz",
       "integrity": "sha512-UOaz/wFowmoh2G6Mr9gw60B1mm0MzUtm6Ic8G2yM1Le6gyj5Loi/N+O5mocugRGY+8OeeKmkMmbxNqUCq3B4Sg==",
       "dev": true,
-      "license": "MIT",
```
Comment (mathiazom) [inline package-lock.json:8047]: What? 🤠

Comment (anemne) [reply package-lock.json:8047]: 😎

File: studio/schemas/documents/siteSettings/brokenLinks.ts [hunk 2]
```diff
@@ -31,11 +30,11 @@ const requiredIfDestinationType = (
   return true;
 };
 
-export const redirectId = "redirect";
+export const brokenLinksID = "brokenLinks";
```
Comment (mathiazom) [inline studio/schemas/documents/siteSettings/brokenLinks.ts:33]:
```suggestion
export const brokenLinkID = "brokenLink";
```

File: package-lock.json (hunk 1 above)
Comment (mathiazom) [reply package-lock.json:8047]: Why did you remove it?

Comment (anemne) [reply package-lock.json:8047]: Removed what?

**Review by mathiazom — APPROVED** [review]

File: studio/lib/queries/brandAssets.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import { groq } from "next-sanity";
 
-import { brandAssetsID } from "studio/schemas/documents/brandAssets";
+import { brandAssetsID } from "studio/schemas/documents/siteSettings/brandAssets";
```
Comment (christinaroise) [inline studio/lib/queries/brandAssets.ts:3]:
```suggestion
import { brandAssetsID } from "studio/schemas/documents/siteSettings";
```

File: studio/lib/queries/companyDetails.ts [hunk 4]
```diff
@@ -1,6 +1,6 @@
 import { groq } from "next-sanity";
 
-import { companyInfoID } from "studio/schemas/documents/companyInfo";
+import { companyInfoID } from "studio/schemas/documents/admin/companyInfo";
```
Comment (christinaroise) [inline studio/lib/queries/companyDetails.ts:3]:
```suggestion
import { companyInfoID } from "studio/schemas/documents/admin";
```

File: studio/lib/queries/compensations.ts [hunk 5]
```diff
@@ -0,0 +1,5 @@
+import { groq } from "next-sanity";
```
Comment (christinaroise) [inline studio/lib/queries/compensations.ts:1]: studio/lib/queries/compensations.ts --> studio/lib/queries/specialPages.ts

File: studio/lib/queries/pages.ts [hunk 6]
```diff
@@ -87,10 +87,6 @@ export const BLOG_PAGE_QUERY = groq`
   *[_type == "blog" && slug.current == $slug][0]
```
Comment (christinaroise) [inline studio/lib/queries/pages.ts:87]: I think this file can be renamed to page.ts

**Review by christinaroise — APPROVED** [review]

File: studio/lib/interfaces/socialMedia.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import { soMeLinksID } from "studio/schemas/documents/socialMediaProfiles";
+import { soMeLinksID } from "studio/schemas/documents/siteSettings/socialMediaProfiles";
```
[the remark this answers is not in the document]
Comment (anemne) [reply studio/lib/interfaces/socialMedia.ts:1]: In the documents section I believe it's better to keep a separate file for each siteSettingelement. Combining all settings into a single file could become more difficult to manage.

**Pull request merged**